### PR TITLE
allow specifying regname for GinBrowse ++yank

### PR DESCRIPTION
**Pull request by atusy**
SSIA

- 👍 add optional reg argument to yank()
- 👍 allow specifying register to GinBrowser ++yank
- 🌿 test yank() with a explicit register
- 📝 document GinBrowse ++yank={regname}

### CODE REVIEW

File: denops/gin/command/browse/command.ts [hunk 1]
```diff
@@ -15,14 +15,14 @@ import { yank } from "../../util/yank.ts";
 
 export type ExecOptions = Omit<Options, "cwd" | "aliases"> & {
   worktree?: string;
-  yank?: boolean;
+  yank: string | boolean;
```
Comment (lambdalisue) [inline denops/gin/command/browse/command.ts:18]: Please don't make it required.

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: denops/gin/command/browse/command.ts (hunk 1 above)
Comment (atusy) [reply denops/gin/command/browse/command.ts:18]: got it

Comment (atusy) [reply denops/gin/command/browse/command.ts]: oops I made wrong commit

Comment (atusy) [reply denops/gin/command/browse/command.ts]: should be fine now

**Review by lambdalisue — APPROVED** [review]
LGTM　Thanks 🎉

**Pull request merged**